boost/multiprecision/uintwide_t_backend.hpp

```diff
@@ -90,24 +90,24 @@
     template<typename UnsignedIntegralType,
              typename std::enable_if<(   (std::is_integral<UnsignedIntegralType>::value == true)
                                       && (std::is_unsigned<UnsignedIntegralType>::value == true))>::type const* = nullptr>
-    constexpr uintwide_t_backend(UnsignedIntegralType u) : m_value(representation_type(std::uint64_t(u))) { }
+    constexpr uintwide_t_backend(UnsignedIntegralType u) : m_value(representation_type(std::uint64_t(u))) { } // NOLINT(google-explicit-constructor,hicpp-explicit-conversions)
```

Comment:
Curious: when you mark the c'tor explicit, what problems arise? Given this is a template, I would have thought implicit conversions would be unlikely. (I am not much of an expert of the language though.)